### ISSUE TITLE
Moto SensorHub

### ISSUE THREAD

**Issue by penn5** [racer123/android_device_nokia_PLE#1]
I think you're still using the Moto SensorHub, which will (almost) certainly cause sensors not to work. Please correct me if I'm wrong, as I haven't searched properly, but I noticed this file: https://github.com/racer123/android_device_nokia_PLE/blob/lineage-15.1-64/include/hardware/mot_sensorhub_stml0xx.h which seems to suggest that sensors are still using the Cedric code.

**Comment by penn5** [comment]
Just looked again, I see you deleted the Moto sensor hal, but you forgot to delete this inclusion.

**Comment by tboHammu** [comment]
Yes, I've still got many unused files in the tree. Sensors are working fine, though.
